code/src/Components/AlbumCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import data from '../data.json';
+
+import { AlbumInfo } from './AlbumInfo';
+
+// The actual div containing all information such as image, artist name, album name and the correct URL-links
+export const AlbumCard = () => {
+  return (
+    <>
+      {data.albums.items.map(album => { // Mapping through the album array to get album-name, images, and the artist array.
+        return <AlbumInfo 
+          key={album.name}
+          image={album.images[0].url} 
+          name={album.name} 
+          artist={album.artists.map(artist => { // Mapping through the artist array for when having several artists
```

Comment:
When our return statement is on several lines, we should wrap it in parentheses:
```
return (
  //code
  //code
)
```